dashed/models.py

```diff
@@ -285,6 +285,10 @@ def grains(self):
                 Grain('Time Column', '{col}'),
                 Grain('week', "date_trunc('week', CAST({col} AS DATE))"),
                 Grain('month', "date_trunc('month', CAST({col} AS DATE))"),
+                Grain("week_ending", "date_add('day', 5, "
+                      "date_trunc('week', date_add('day', 1, CAST({col} AS DATE))))"),
+                Grain("week_start", "date_add('day', -1, "
```

Comment:
the first argument shows as the label in the dropdown, `week_start_monday` `week_ending_sunday`  would be more clear.